apps/src/utils.js

```diff
@@ -235,7 +235,13 @@ export function isInfiniteRecursionError(err) {
 export function browserSupportsCssMedia() {
   var styleSheets = document.styleSheets;
   for (var i = 0; i < styleSheets.length; i++) {
-    var rules = styleSheets[i].cssRules || styleSheets[i].rules;
+    var rules;
+    try {
+      rules = styleSheets[i].cssRules || styleSheets[i].rules;
+    } catch (e) {
+      // Some versions of Chrome give an error when accessing cssRules.
+      return false;
```

Comment:
The comment on this function suggests it's an old IE9 guard, but we haven't supported IE9 for a while so I assume the `return false` path here has not been tested for a long time.  Do we know the ramifications of this?

Reply:
we do not. perhaps we can get GT or whoever can repro to test this against my machine or an adhoc before we ship it.

Reply:
I did a little digging and it looks like we just do some extra visualization resizing (probably safe?) if we don't have access to media rules.